users/contrib.rst

```diff
@@ -78,6 +78,18 @@ Command Line Tools
   Syncthing Cli - a simple command line tool for getting status and performing basic operations from
   the shell / terminal without need of a web browser.
 
+- `syncthing-graph <https://gitlab.com/andrea-trentini/syncthing-graph>`_
+
+  Very simple graph (dot format) generator for Syncthing ``config.xml``.
+
+- `syncthing-map <https://github.com/wsw70/syncthing-map>`_
+
+  A utility to map Syncthing devices and shared folders.  Generates a
```

Comment:
```suggestion
  A cross-platform utility to map Syncthing devices and shared folders.  Generates a
```

What do you think? The other one seems Linux-only (not sure about macOS) while this one seems to compile and work everywhere.